status/status.go

```diff
@@ -22,6 +28,21 @@ import (
 	"go.dedis.ch/onet/v3/network"
 )
 
+const prometheusTemplate = `
+# HELP voting_conodes_status voting global conodes status: X is the number of conodes; X >= 0.66 OK, 0 < X < 0.66 KO
+# TYPE voting_conodes_status gauge
+voting_conodes_status{} {{.Connectivity}}
+# HELP voting_conode_status voting conode status: 1=OK, 0=KO
+# TYPE voting_conode_status gauge
+{{- range $conode, $status := .Matrix -}}
+{{ if eq $conode $.Self }}
+voting_conode_status{conode="{{ $conode}}", critical="true"} {{$status}}
+{{ else }}
+voting_conode_status{conode="{{ $conode}}", critical="false"} {{$status}}
+{{ end }}
```

Comment:
Even if it was not requested by EPFL, I would add a field to expose the timestamp at which we last checked connectivity.

It is a typical Prometheus pattern and allows one to detect if the connectivity check stopped for some reason, and we keep serving the same old vlaues.